src/Menu/Menu.js

```diff
@@ -148,9 +148,10 @@ class Menu extends Component {
      */
     valueLink: PropTypes.object,
     /**
-     * The width of the menu. If not specified, the menu's width
-     * will be set according to the widths of its children, using
-     * proper keyline increments (64px for desktop, 56px otherwise).
+     * The width of the menu in string ("200px", "50vw") or number (200, 300)
```

Comment:
Hi @mbrookes I specifically indicated `50vw` because that's what I think where the confusion might come in. I actually tried using `50%` but `<Menu>` doesn't allow it so I tried `50vw` instead and it worked. So I thought specifying in the Docs that it accepts only `px` and `vw` might greatly help people who have the same problem.